packages/plugin-bundler-webpack/src/webpack.config.js

```diff
@@ -2,77 +2,128 @@ import path from "path";
 
 import webpack from "webpack";
 import ExtractTextPlugin from "extract-text-webpack-plugin";
+import HtmlWebpackPlugin from "html-webpack-plugin";
+import autoprefixer from "autoprefixer";
 
-module.exports = (config: PhenomicConfig) => ({
-  entry: {
-    [config.bundleName]: [
-      process.env.PHENOMIC_ENV !== "static" &&
-        require.resolve("webpack-hot-middleware/client"),
-      process.env.PHENOMIC_ENV !== "static" &&
-        require.resolve("react-hot-loader/patch"),
-      "./App.js"
-    ].filter(item => item)
-  },
-  output: {
-    publicPath: "/", // @todo make this dynamic
-    path: path.isAbsolute(config.outdir)
-      ? config.outdir
-      : path.join(process.cwd(), config.outdir),
-    ...(process.env.PHENOMIC_ENV !== "static"
-      ? {
-          filename: "phenomic/[name].js",
-          chunkFilename: "phenomic/[name].chunk.js"
+module.exports = (config: PhenomicConfig) => {
+  const isStatic = process.env.PHENOMIC_ENV === "static";
+  const isProduction = process.env.NODE_ENV === "production";
+
+  const baseConfig = require("react-scripts/config/webpack.config." +
```

Comment:
I'm a bit worried to rely on a react repo for the webpack config (they are not coupled in Phenomic)